docs/source/1.0/spec/core/model.rst

```diff
@@ -618,6 +618,13 @@ SHOULD represent sets as a custom set data structure that can interpret value
 hash codes and equality, or alternatively, store the values of a set data
 structure in a list and rely on validation to ensure uniqueness.
 
+.. rubric:: Set member ordering
+
+Sets returned by generated tooling, such as the deserialized response to a
+client request, MUST be insertion ordered. Sets accepted as input by generated
+tooling, such as in a client request, SHOULD be ordered unless requiring an
+ordered set would be overly burdensome or conflict with language idioms.
```

Comment:
Should these languages just use lists if they can't maintain order?

Reply:
It depends on whether the unique property or the ordered property is more important. I mentioned this offline, but I don't think it's particularly ergonomic in any language to require an ordered set as an input.

Reply:
I think the order would be very important if a service wants a set of unique items but order provides precedence. If in some languages it can't be relied on, then it's not guaranteed that the service will work interoperably across all of its clients, leaving the service to use a list instead, and now that uniqueItems is being removed, to have to implement validation and making the list unique manually rather than relying on a framework feature.